pkgs/development/compilers/gmqcc/default.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  pkgs,
+  stdenv,
+  fetchgit,
+  ...
+}:
+
+stdenv.mkDerivation rec {
+    pname = "gmqcc";
+    version = "unstable-2021-07-09";
+
+    src = fetchgit {
```

Comment:
Please use `fetchFromGitHub` instead. This will use tarballs inside of the `git clone`, and it is much more efficient in terms of bandwidth usage.